packages/workspace/src/generators/new/generate-workspace-files.ts

```diff
@@ -134,11 +134,7 @@ function createNxJson(
 function createFiles(tree: Tree, options: NormalizedSchema) {
   const formattedNames = names(options.name);
   const filesDirName =
-    options.preset === Preset.AngularStandalone ||
-    options.preset === Preset.ReactStandalone ||
-    options.preset === Preset.NodeServer
-      ? './files-root-app'
-      : options.preset === Preset.NPM || options.preset === Preset.Core
+    options.preset === Preset.NPM || options.preset === Preset.Core
```

Comment:
Could you please explain what's changing here?

So we originally had a separate set of files we use for standalone apps. Now when people choose a standalone preset.. they get the integrated file set.

I'm guessing it became either identical or so similar we don't need a separate set after we moved `tsconfig.base.json` out. Is the integrated file set now identical to what the standalone file set used to be minus tsconfig.base.json?

Reply:
yes. those raw files look the same. but I revert this change for now. this pr get too big.